handlers/generic_file_upload_ce_attachment_v1/README.md

```diff
@@ -0,0 +1,46 @@
+# Kinetic Request CE Attachment Upload With API
+    This handler is used to copy an attachment from one submission to another.
```

Comment:
In the node xml, you described this handler as `This handler is used to upload an attachment from a submission to a third party using a generic API`